elastic/beats
# Fail integration tests if system requirements are not met

## Proposed commit message



If a system running integration tests doesn't meet the requirements, fail the test instead of passing while skipping all test cases.

Currently, when run on systems without the requirements the tests will be marked as passed, while not actually running any test cases.  This could cause any real failures to be missed, especially in CI, where it's less likely that anyone will look at the logs when the build has been marked as passing.

It's better to fail on systems that don't have the requirements so that such systems can be investigated and fixed, or explicitly skipped at a higher level.

## Checklist




## How to test this PR locally



You can temporarily rename your `docker-compose` executable, so it won't be found by the tests. Then run `mage integTest`, and see that the tests will now fail because of the missing requirement, (and then restore the docker-compose filename so that it will work in the future).
